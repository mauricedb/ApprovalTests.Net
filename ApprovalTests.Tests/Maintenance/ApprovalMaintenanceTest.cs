using ApprovalTests.Maintenance;
using ApprovalUtilities.Utilities;
using NUnit.Framework;

namespace ApprovalTests.Tests.Maintenance
{
	[TestFixture]
	public class ApprovalMaintenanceTest
	{
		[Test]
		public void FindAbandonedApprovalFiles()
		{
			using (var t = new TempFile(PathUtilities.GetAdjacentFile("DeletedClass.AbandoneMethod.approved.txt")))
			{
				t.WriteAllText("Llewellyn was here");
				var path = PathUtilities.GetDirectoryForCaller();
				var list = ApprovalMaintenance.FindAbandonedFiles(path);
				Approvals.VerifyAll("Abandoned Files:",list, f=>f.Name);
			}
		}
	}
}