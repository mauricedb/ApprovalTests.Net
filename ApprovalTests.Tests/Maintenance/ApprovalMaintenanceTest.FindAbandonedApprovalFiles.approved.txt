Abandoned Files:

DeletedClass.AbandoneMethod.approved.txt
